lib/index.d.ts

```diff
@@ -146,6 +146,7 @@ declare namespace sharp {
     const kernel: KernelEnum;
     const fit: FitEnum;
     const bool: BoolEnum;
+    const preset: PresetEnum
```

Comment:
How about `WebPPresetEnum` (or similar, the capitalisation looks a bit odd) to clarify this applies only to WebP images?

Reply:
There don't seem to be any other format-specific enums that I can spot, but as a model yes that makes sense.  What if there were someday JpegXL-presets ... 

The WebP options are named `WebpOptions` so I'll rename this to `WebpPresetEnum` which also looks a little better.

Reply:
This update has been pushed (the rename).  Ready for review again.